### PR TITLE
contrib/hyprland: new package (0.29.1)

### PR DESCRIPTION
**Pull request by Froggo8311**

### CODE REVIEW

File: contrib/hyprland/template.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+makedepends = [
+    "cairo-devel",
+    "hwdata-devel",
+    "libavcodec",
+    "libavformat",
+    "libavutil",
+    "libdisplay-info-devel",
+    "libdrm-devel",
+    "libgbm-devel",
+    "libgles2",
```
Comment (nekopsykose) [inline contrib/hyprland/template.py:22]:
```suggestion
    "ffmpeg-devel",
    "libdisplay-info-devel",
    "libdrm-devel",
    "libgbm-devel",
```

Comment (Froggo8311) [reply contrib/hyprland/template.py]: That also works, thanks. I forgot to remove gles after debugging and didn't know there was an `ffmpeg-devel` package that provided the others

**Comment by q66** [comment]
no

**Comment by jiripospisil** [comment]
Why?

**Comment by jiripospisil** [comment]
Huh? You're the one who opened the PR in the first place. And yeah, if you're going to exclude one of the most popular Wayland compositor, something more than "no" would have been great.

**Comment by q66** [comment]
hyprland's case is a very particular combo of "made by very loudly bigoted clowns" and "is a terrible codebase whose author displays a total lack of fundamentals"

neither thing is particularly uncommon in foss/tech (sadly) and it's not possible to check/verify everything, but since we're dealing with a particular combo of these where it's really visible and obvious, i see no reason to let it in

i think you're also overestimating its actual popularity (not that it would have any effect on the decision)

so there you go

**Pull request closed without merging**